### ISSUE TITLE
[BUG] web: too restrictive URL format for machine learning server

### ISSUE THREAD

**Issue by tbleiker** [immich-app/immich#4525]
### The bug

In my docker setup, I use `http://machine-learning.immich_default:3003` as the url for my machine learning server. That perfectly works when set in docker compose. However, the web interface does not like such urls and gives me the following error: 

```
ERROR machineLearning.url must be a URL address
(Immich Server Error)
```

### The OS that Immich Server is running on

Docker on Debian Bookworm

### Version of Immich Server

v1.82.0

### Version of Immich Mobile App

v1.82.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
...
```

### Your .env content

```Shell
...
```

### Reproduction steps

```bash
1. Log in to the web interface
2. Head over to settings
3. Open "Machine Learning Settings"
4. Enter something like `http://machine-learning.immich_default:3003` as url
5. Hit `Save`
6. Enjoy the error message
```

**Comment by bo0tzz** [comment]
This was just fixed in #4517

**Comment by tbleiker** [comment]
Wow, that was fast...! 🏎 🤣
